build/unit_tests/test_codegen_helper.py

```diff
@@ -42,6 +42,7 @@
         'type': 'ViSession',
         'numpy': False,
         'complex_type': None,
+        'array_dimension': 1,
```

Comment:
Adding this to these dicts does absolutely nothing for our testing, because these tests don't call `add_all_function_metadata()`, even indirectly.

Is there some other usage that I'm missing?